tests/Unit/PointwiseFunctions/Elasticity/ConstitutiveRelations/Test_Tags.cpp

```diff
@@ -3,15 +3,55 @@
 
 #include "tests/Unit/TestingFramework.hpp"
 
+#include <cstddef>
 #include <string>
 
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "Parallel/ParallelComponentHelpers.hpp"
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/ConstitutiveRelation.hpp"
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/IsotropicHomogeneous.hpp"
 #include "PointwiseFunctions/Elasticity/ConstitutiveRelations/Tags.hpp"
 
+namespace Elasticity {
+namespace ConstitutiveRelations {
+
 namespace {
-struct SomeConstitutiveRelation;
+struct Provider {
+  static constexpr size_t volume_dim = 3;
+  using constitutive_relation_type = IsotropicHomogeneous<volume_dim>;
+  static constitutive_relation_type constitutive_relation() { return {1., 1.}; }
```

Comment:
I don't think there's any way for these to get reordered in this test, but I think it's good testing philosophy generally to pass in different numerical arguments so that it makes sure that the values ended up in the parameters you intended.